File: Classes/WMISet/WMISet/Examples/delphi/ProcessControl/FrmSelectColumnsU.pas
unit FrmSelectColumnsU;

interface
{$I define.inc}

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms,
  Dialogs, StdCtrls, ExtCtrls;

type
  TFrmSelectColumns = class(TForm)
    chbImageName: TCheckBox;
    chbIOOther: TCheckBox;
    chbPID: TCheckBox;
    chbIOOtherBytes: TCheckBox;
    chbCpuUsage: TCheckBox;
    chbPagedPool: TCheckBox;
    chbCpuTime: TCheckBox;
    chbNonPagedPool: TCheckBox;
    chbMemoryUsage: TCheckBox;
    chbBasePriority: TCheckBox;
    chbPeakMemoryUsage: TCheckBox;
    chbHandleCount: TCheckBox;
    chbPageFaults: TCheckBox;
    chbThreadCount: TCheckBox;
    chbIOReads: TCheckBox;
    chbVirtualMemory: TCheckBox;
    chbIOReadBytes: TCheckBox;
    chbSessionId: TCheckBox;
    chbIOWrites: TCheckBox;
    chbStartedAt: TCheckBox;
    chbIOWriteBytes: TCheckBox;
    chbFullPath: TCheckBox;
    Bevel1: TBevel;
    btnOk: TButton;
    btnCancel: TButton;
    procedure FormCreate(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  FrmSelectColumns: TFrmSelectColumns;

implementation

{$R *.dfm}

procedure TFrmSelectColumns.FormCreate(Sender: TObject);
begin
  ClientWidth := btnCancel.Left + btnCancel.Width + 7; 
  ClientHeight := btnCancel.Top + btnCancel.Height + 7; 
end;

end.

File: Classes/WMISet/WMISet/Examples/cbuilder/ProcessControl/FrmSelectColumnsU.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "FrmSelectColumnsU.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma resource "*.dfm"
TFrmSelectColumns *FrmSelectColumns;
//---------------------------------------------------------------------------
__fastcall TFrmSelectColumns::TFrmSelectColumns(TComponent* Owner)
        : TForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TFrmSelectColumns::FormCreate(TObject *Sender)
{
  ClientWidth = btnCancel->Left + btnCancel->Width + 7; 
  ClientHeight = btnCancel->Top + btnCancel->Height + 7; 
}
//---------------------------------------------------------------------------
